src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py

```diff
@@ -150,16 +150,12 @@ def _cc_close(self) -> None:
             del self.logger
             self.logger = None
 
-    def _cc_send(self, msg: MessageType, raw: bool = False, **kwargs) -> None:
+    def _cc_send(self, msg: MessageType, **kwargs) -> None:
```

Comment:
@sebastianpfischer 
For all CAN connectors i see that we pass **kwargs.

Shouldn't msg not be a dict with to fields?
I would expect "data" and "remote_id"

Reply:
No, we considered that msg is something that can be sent raw. Which is the minimal allowed. the kwargs contain the remote_id

Reply:
I'm wondering if the remote_id shouldn't be specified as explicit kwarg (the additional kwargs can still be kept)

Reply:
Done